docs/source/api.rst

```diff
@@ -5,12 +5,16 @@ cuML API Reference
 Module Configuration
 ====================
 
+.. _output-data-type-configuration:
+
 Output Data Type Configuration
 ------------------------------
 
  .. automethod:: cuml.common.memory_utils.set_global_output_type
  .. automethod:: cuml.common.memory_utils.using_output_type
 
+.. _verbosity-levels:
```

Comment:
These become subsections, right? Not top level sections?

Reply:
No this is just creates a named reference marker that we can then refer to anywhere in the documentation with `ref:verbosity-levels`.